lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -36,12 +36,12 @@
 		</div>
 	</div>
 
-	<%= fields %>
+	<%= HtmlUtils.HTML(fieldsHtml) %>
```

Comment:
This should be ``HtmlUtils.ensureHtml``, if I remember correctly. I think this will double escape as it is written.